package.json

```diff
@@ -1775,10 +1775,10 @@
                     "description": "Allow for connecting the Interactive window to a https Jupyter server that does not have valid certificates. This can be a security risk, so only use for known and trusted servers.",
                     "scope": "resource"
                 },
-                "jupyter.enablePlotViewer": {
+                "jupyter.generateSVGPlots": {
```

Comment:
I didn't put a deprecation on the old setting here, as with the meaning of the setting reversed it didn't seem needed. Before it was defaulted to true (SVGs generated), now it's defaulted to false (no SVGs generated). Before you would only change it to false if you didn't want the SVGs and that's now the default so we don't need to worry about customers who set this to false before.

Note that it only reads as "false" in the left side of the change here as I changed it with the initial PNG change this week. It was defaulted to true in the last real release.